vega_sim/api/trading.py

```diff
@@ -240,7 +240,7 @@ def cancel_order(
     wallet_name: str,
     wallet: Wallet,
     market_id: str,
-    order_id: str,
+    order_id: Optional[str],
```

Comment:
Whats the purpose of making this optional?

Reply:
If you pass as None it cancels everything on the market